app/Http/Requests/Three/PostRequest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Rogue\Http\Requests\Three;
+
+use Rogue\Http\Requests\Request;
+
+class PostRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'status' => 'in:pending,accepted,rejected',
```

Comment:
Should we add any validation to the `caption` field? Perhaps `string|max:140`?

Reply:
great call, @DFurnes ! I updated this and also the `testValidationUpdatingAPost` test as well to incorporate this.